config/routes.rb

```diff
@@ -16,6 +16,12 @@
     root to: "auth/sessions#new"
   end
 
+  get '/installer/options', to: 'installer#options'
+  get '/installer/master', to: 'installer#master'
+  get '/nodes/add', to: 'nodes#add'
+  get '/nodes', to: 'nodes#list'
+  get '/config', to: 'config#index'
```

Comment:
Are we using these as "example" pages? I'm not sure about what purpose each one serves.